README.md

```diff
@@ -167,6 +167,38 @@ NSImage(systemSymbol: .cCircle)
 NSImage(systemSymbol: SFSymbol.eCircleFill, accessibilityDescription: "some.description")
 ```
 
+## Localization
+
+SFSymbols can come with multiple different localizations. `SFSafeSymbols` exposes localization the following way:
+
+- Implicit localization: When using an `SFSymbol`, it gets automatically localized to the user's current locale - nothing to do on your part.
+
+- Explicit localization: `SFSafeSymbols` lets you access a symbol's localized versions as follows:
+
+  ```swift
+  // 1. Static localization:
+  let a = SFSymbol.character.ar // corresponds to "character.ar"
+  let b = SFSymbol.character.zh // corresponds to "character.zh"
+  let c = SFSymbol.character.rtl // doesn't compile: "character.rtl" doesn't exist
+  // a, b have type SFSymbol
+  
+  // 2. Dynamic localization:
+  SFSymbol.character.availableLocalizations // [.ar, .he, .hi, .ja, .ko, .th, .zh, .zhTraditional]
+  let a = SFSymbol.character.localized(to: .ar)
+  let b = SFSymbol.character.localized(to: .rtl)
+  // a, b have type SFSymbol?
+  ```
+
+  Static localization only exposes the localizations which are actually available, so you cannot accidentally localize a non-localizable symbol.
+
+  Dynamic localization is useful however when dealing with an array of `LocalizableSFSymbols` which all have different localization information.
```

Comment:
Did you read the second comment? (_"Also `localization information` sounds a bit weird. Maybe better use `available localizations`."_) Because if you don't have anything against it, I would suggest to change the wording.